src/code42cli/cmds/users.py

```diff
@@ -0,0 +1,84 @@
+import click
+from pandas import DataFrame
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.click_ext.options import incompatible_with
+from code42cli.errors import Code42CLIError
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import DataFrameOutputFormatter
+
+
+@click.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def users(state):
+    """Manage users within your Code42 environment"""
+    pass
+
+
+org_uid_option = click.option(
+    "--org-uid",
+    required=False,
+    type=str,
+    default=None,
+    help="Limit users to only those in the organization you specify. ",
+)
+role_name_option = click.option(
+    "--role-name",
+    required=False,
+    type=str,
+    default=None,
+    help="Limit results to only users having the specified role.",
+)
+active_option = click.option(
+    "--active", is_flag=True, help="Limits results to only active users.", default=None,
+)
+inactive_option = click.option(
+    "--inactive",
+    is_flag=True,
+    help="Limits results to only deactivated users.",
+    cls=incompatible_with("active"),
+)
+
+
+@users.command(name="list")
+@org_uid_option
+@role_name_option
+@active_option
+@inactive_option
+@format_option
+@sdk_options()
+def list_users(state, org_uid, role_name, active, inactive, format):
+    """List users in your Code42 environment."""
+    if inactive:
+        active = False
+    if role_name:
```

Comment:
Could be a ternary here
`role_id = _get_role_id(state.sdk, role_name) if role_name else None`